debian/changelog

```diff
@@ -1,3 +1,7 @@
+sibench (1.1) buster; urgency=low
```

Comment:
I'm thinking, maybe we should stay on x.y.z versioning, making this one 1.1.0, thoughts?

Reply:
Oh yeah, good catch